### DISCUSSION TITLE
NEW아이콘을 디자인 해서 내놓을수는 없을가요?

### DISCUSSION THREAD

**Discussion by bank365** [pronist/hELLO#238] [Ideas]
안녕하세요.?
기본적으로 제공되는 NEW 아이콘이
스킨의 디자인을 해친다면, 첨부 이미지처럼 헬로스킨에 맞는 NEW마크로 디자인해서 보일수는 없을가요?
새로운 게시글을 보여준다는 의미에서 NEW마크는 개인적인 의견이지만... 필요할것 같아서요.

![Screenshot 2024-06-30 at 14 01 02](https://github.com/pronist/hello/assets/172935667/99a4068d-418b-4060-bd83-26f4c3b7a3b5)

**Comment by pronist** [comment]
@bank365 예시 사진이 프라치노 공책이면 유료 스킨도 구입해서 사용하시는 것 같은데... 개인적으로 필요하다고 느껴지신다면 구입하신 스킨을 쓰시는 것이 나아보입니다.

**Reply by bank365** [reply]
공책스킨이신건 어케 아셧대요 ;;;; ㄷㄷ;;;;
프라치노 계열 스킨들은 현재 개발자님이 잠수 및 오래되서인지 일부 기능서 오류를 뿜고 있습니다.
그러던 와중에 예전에 배포하셨던 헬로라는 스킨을 우연히 다시 마주했고 꾸준한 업데이트와 절제된 디자인을 보고 반했습니다.
NEW마크를 지원 안하시는걸로 알고, 저도 이 사항에 대해 더이상 언급 안하겠습니다.
답변 감사합니다^^
(뒷말: 헬로스킨이 유료배포였다면, 결재 했을 겁니다.!!! 제 기억 상으론 예전에 프라치노님이 개발자님 스킨보고 디자인 침해라고 정정 요청을 드렸던 걸로 아는데, 그것 때문에 제가 불편을 드린 건 아닌가 싶네요. 미안합니다 ㅠㅠ 스샷 일부만 보고 공책 스킨 인 걸 알 줄은 미처 몰랐습니다.ㅠㅠ. 지금도 다른 유료 스킨 구입해서 이용중이긴 한데... 제 취향이 헬로 스킨 중의 하나라...)
